src/App.js

```diff
@@ -27,47 +22,16 @@ function App() {
 
   useEffect(mountFirebaseAuth, []);
 
-  let shownScreen = signedIn ? 
-                    <div>
-                      <h1>Welcome</h1>
-                      <SignOut />
-                    </div> : 
-                    <Login />;
+  let shownScreen = signedIn ? <Validator /> : <Login />;
```

Comment:
Ok, I tried to implement this! In doing so, I introduced  bug- if the use presses 'login' without proper credentials, they are sent to a blank /validator page (because they aren't allowed to see the page). We can re-assess this later because it seems fixable, but I will merge it as is for now.

Reply:
Sounds good